lang/German.ini

```diff
@@ -185,6 +185,11 @@ MARIO_THEME = "Mario"
 ODYSSEY_THEME = "Odyssey"
 FILE_SELECT_THEME = "Dateiauswahl"
 
+[CHAT_DISPLAY]
+TEXT_OPACITY = "Chat-Textopazität"
+BACKGROUND_OPACITY = "Opazität des Chat-Hintergrunds"
+CHAT_SCALE = "Chat-Skala"
```

Comment:
I think "Chat-Größe" would work better here

Reply:
if you directly translate it, it'll give you "chat size"
while "Chat-Skala" does work, it seems a little unfitting for the context, which is why "Chat-Größe" makes more sense here